Gem/Settings/Scenes/SecurityScene.swift

```diff
@@ -6,65 +6,46 @@ import LocalAuthentication
 import Style
 
 struct SecurityScene: View {
-    
-    let model: SecurityViewModel
-    @State public var authenticationEnabled: Bool
-    @State public var authenticationInProgress: Bool = false
-    @State private var isPresentingErrorMessage: String?
+    @State private var model: SecurityViewModel
 
-    init(
-        model: SecurityViewModel
-    ) {
+    init(model: SecurityViewModel) {
         self.model = model
-        self.authenticationEnabled = model.authenticationEnabled
     }
     
     var body: some View {
         List {
-            HStack {
-                Toggle(
-                    model.authenticationTitle,
-                    isOn: $authenticationEnabled
+            Toggle(model.authenticationTitle, isOn: $model.isEnabled)
+            .toggleStyle(AppToggleStyle())
+
+            NavigationLink(value: model.lockPeriodModel.selectedOption) {
+                ListItemView(
+                    title: model.lockPeriodModel.title,
+                    subtitle: model.lockPeriodModel.selectedOption.title
                 )
-                .toggleStyle(AppToggleStyle())
-            }
-        }
-        .onChange(of: authenticationEnabled) {
-            Task {
-                let context = LAContext()
-                do {
-                    guard try await context.evaluatePolicy(.deviceOwnerAuthentication,
-                        localizedReason: Localized.Settings.Security.authentication
-                    ) else {
-                        return
-                    }
-                    do {
-                        try model.changeEnableBiometrics(value: authenticationEnabled, context: context)
-                    } catch {
-                        DispatchQueue.main.async {
-                            isPresentingErrorMessage = error.localizedDescription
-                        }
-                    }
-                } catch {
-                    DispatchQueue.main.async {
-                        authenticationEnabled = model.authenticationEnabled
-                    }
-                }
             }
         }
-        .alert(item: $isPresentingErrorMessage) {
+        .onChange(of: model.isEnabled, onToggleEnable)
+        .alert(item: $model.isPresentingError) {
             Alert(title: Text("Transfer Error"), message: Text($0))
```

Comment:
```suggestion
            Alert(title: Text("Transfer Error"), message: Text($0))
```

different title for security screen?